hail/python/hail/backend/spark_backend.py

```diff
@@ -268,7 +273,18 @@ def hail_package(self):
     def utils_package_object(self):
         return self._utils_package_object
 
+    def _rpc(self, action, payload) -> Tuple[bytes, str]:
```

Comment:
This can live on `Py4JBackend`, right?

Reply:
I see that you depend on _backend_server. You could pass the `_jbackend` into `Py4JBackend`'s `__init__` and then you can construct `_backend_server` in `Py4JBackend`.

Reply:
Good call. I took it several steps further and also lifted a bunch of other duplication into `Py4JBackend`. Should be limited to just commit `aa3ffcc` if you want to look at that separately.